types/signable.go

```diff
@@ -3,13 +3,17 @@ package types
 import (
 	"github.com/cometbft/cometbft/crypto/bls12381"
 	"github.com/cometbft/cometbft/crypto/ed25519"
+	"github.com/cometbft/cometbft/crypto/sr25519"
 	cmtmath "github.com/cometbft/cometbft/libs/math"
 )
 
 // MaxSignatureSize is a maximum allowed signature size for the Proposal
 // and Vote.
-// XXX: secp256k1 does not have Size nor MaxSize defined.
-var MaxSignatureSize = cmtmath.MaxInt(ed25519.SignatureSize, cmtmath.MaxInt(bls12381.SignatureLength, 64))
+// XXX: secp256k1 does not have max signature size defined.
+var MaxSignatureSize = cmtmath.MaxInt(
```

Comment:
The problem of this approach is that blocks produced using different signing algorithms will have different maximum payload size. We need to re-think this way of computing the maximum payload size, as it depends on the signing algorithm we are using...